c10/macros/cmake_macros.h.in

```diff
@@ -7,7 +7,7 @@
 #cmakedefine C10_BUILD_SHARED_LIBS
 #cmakedefine C10_USE_GLOG
 #cmakedefine C10_USE_GFLAGS
-#cmakedefine C10_DISABLE_NUMA
+#cmakedefine USE_NUMA @USE_NUMA@
```

Comment:
So, if I understand correctly, you renamed the macro here so you wouldn't have to rename all the `USE_NUMA` sites?

Reply:
(I've corrected this line) There is one place that uses `C10_DISABLE_NUMA` and I have changed that accordingly.